src/store/reducers/index.js

```diff
@@ -1,12 +1,6 @@
-const initialState = {
-    articles: [],
-    notifications: [],
-    followers: [],
-}
+import { combineReducers } from 'redux';
+import articlesReducer from './articlesReducer';
 
-function rootReducer(state = initialState, action){
-
-    return state;
-};
-
-export default rootReducer;
+export default combineReducers({
```

Comment:
The way you have exported the combined reducer here, the store will fail because it expects a constant named root reducer. You can export it like this:
```suggestion
const rootReducer = combineReducers({
articles: articleReducer})
export default rootReducer
```